sphinx_gallery/binder.py

```diff
@@ -103,12 +111,26 @@ def gen_binder_rst(fpath, binder_conf, gallery_conf):
     binder_conf = check_binder_conf(binder_conf)
     binder_url = gen_binder_url(fpath, binder_conf, gallery_conf)
 
+    binder_logo = os.path.join(glr_path_static(), 'binder_badge_logo.svg')
+    binder_logo_full_path = os.path.join(
+        target_dir, 'images', 'binder_badge_logo.svg'
+    )
+    if gallery_conf['gallery_dirs'] is None:
+        # Avoid copying file when testing
+        pass
+    else:
+        if not os.path.exists(binder_logo_full_path):
+            shutil.copyfile(binder_logo, binder_logo_full_path)
```

Comment:
and if you use `_static`, no need to `copyfile`

Reply:
Can you clarify? As in I can use the logo in the packaged static file e.g.
```
.. image:: /home/lucy/Documents/sphinx-gallery/sphinx_gallery/_static/binder_badge_logo.svg
``` 
?

Reply:
You're right, that would have saved a lot of time!

Reply:
I would try putting just `/_static/binder_badge_logo.svg` here. If it doesn't work, then doing `os.path.join(os.path.relpath(src_dir, this_file_dir), '_static', 'binder_badge_logo.svg')` or so should work. Basically it already lives in `_static` so we should just reuse it

Reply:
> /_static/binder_badge_logo.svg

that doesn't work but the full path works

Reply:
Sure, whatever path works is fine by me!